ops/model.py

```diff
@@ -760,6 +763,7 @@ def __delitem__(self, key):
 
 
 class ConfigData(LazyMapping):
+    """Configuration data."""
```

Comment:
Shouldn't this have the usual "should not be instantiated by the user" blurb?

Reply:
Sorry, I merged this without checking these comments, but will add these improvements to next branch.

Added a note as you suggested.